sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -275,10 +275,17 @@ def get_a2_df():
             memory = spec['MemoryGiB']
             price = per_cpu_price * cpu + per_memory_price * memory
             spot_price = per_cpu_spot_price * cpu + per_memory_spot_price * memory
-            table.append([instance_type, memory, price, spot_price, region])
-    a2_df = pd.DataFrame(
-        table,
-        columns=['InstanceType', 'MemoryGiB', 'Price', 'SpotPrice', 'Region'])
+            table.append(
+                [instance_type, cpu, memory, price, spot_price, region])
+    a2_df = pd.DataFrame(table,
+                         columns=[
+                             'InstanceType',
+                             'vCPU',
```

Comment:
> 1. Do we need we increment the catalog schema version?

It is a hard decision. I didn't consider incrementing the schema version because then we need to maintain both v1 and v2. And there's a possibility that we need to add more columns (e.g., cpu type, network bandwidth, etc.) to the GCP catalog in the future. The version number can grow fast. 

Adding a new column to the catalog will not break backward compatibility. What may break the compatibility is the code that uses the new column, right? And we can preserve backward compatibility by adding an if statement `if col in df.columns`. I think it's fine to keep the version number when we only add new columns to the catalog.

> 2. Update the catalogs in our [catalogs repo](https://github.com/skypilot-org/skypilot-catalog)?

Yes. I will do that.

Reply:
@romilbhardwaj WDYT?

Reply:
I think we should increment the schema version if we're adding a new column. As you mention, preserving backward compatibility would require adding schema checks like `if col in df.columns`. My worry is these will checks will pile up over time and clutter our code. It would be better to only support the latest schema version in any given commit.

There should be no reason for a user to use an older schema version other than simply because they haven't upgraded yet. Growing version numbers should be okay since its just a matter of convention.

Reply:
After an [offline discussion](https://docs.google.com/document/d/1YZjuA5yElNoBAW_IBqziBYhCMHWEEHyQ-5wTTSp7pUk/edit), we decided to increment the schema version for this column addition. The intent is to minimize bugs from incompatible schema and handling code. Users who may have edited their schemas will need to apply their changes again to the new catalog.

@WoosukKwon - can we add a permanent message saying catalogs have been updated when we download catalogs?Currently its a transient message ("Downloading aws catalog...") that goes away after the download is succesful.
![image](https://user-images.githubusercontent.com/4416605/184972997-67c5c288-665b-4224-abe1-e9f21746295e.png)

Reply:
> @WoosukKwon - can we add a permanent message saying catalogs have been updated when we download catalogs?Currently its a transient message ("Downloading aws catalog...") that goes away after the download is succesful.

Will do that.

Reply:
@romilbhardwaj BTW, what columns besides `vCPU` do you think we should add to the GCP catalog v2? I think it's good to add as more columns as possible even if they are not gonna be used immediately.